### DISCUSSION TITLE
How to expand a command to use out of subcommand in hooks?

### DISCUSSION THREAD

**Discussion by tigerinus** [goreleaser/goreleaser#3359] [Q&A] [answered]
In the `hooks` section I need this to work

```
docker run --rm --user $(id -u {{ .Env.USER }}):$(id -g {{ .Env.USER }}) ...
```

Apparently this is not supported. 

Any workaround?

Thanks.

**Comment by caarlos0** [comment, marked as answer]
Can you share [your config and the output when you try to run it](https://github.com/goreleaser/goreleaser/discussions/2571)?

**Reply by tigerinus** [reply]
Nevermind. I wrapped the command in a bash script and call the script instead.
